src/main/java/uk/gov/justice/probation/courtcaseservice/controller/model/CourtCaseRequest.java

```diff
@@ -75,6 +75,8 @@ public class CourtCaseRequest {
     private final String nationality2;
     private final Boolean awaitingPsr;
     private final PhoneNumber phoneNumber;
+    private final String personId = "d1eefed2-04df-11ec-b2d8-0242ac130002";
```

Comment:
Don't think this assignment should be here 🤔

Reply:
@jhevans I am not sure why this class is in the main package. Not used anywhere except its own test class! I think we should remove this and its test class completely.

Reply:
True, can't see any usages for it. Should be removed.